metis-enrichment/metis-enrichment-service/src/test/java/eu/europeana/enrichment/service/wikidata/WikidataAccessDaoTest.java

```diff
@@ -41,41 +47,129 @@ public void tearDown() throws Exception {}
 
   // TODO JV it is not a good idea to test on real-time data that comes from a external source. This
   // can break our tests if the data is changed.
+  // Answer SG: we can try to split the tests in unit tests and integration tests. However the data structure on wikidata is stable, 
+  // adding new fields or values must not break the code. Otherwise it is clear that we have a bug in the implementation. 
+  //It is good to see such bugs as soon as possible. We may consider to have the same tests implemented both as unit (using local files), and integration (using http requests)tests  
+  /**
```

Comment:
Like we discussed, we had a data change recently that broke our unit tests. Can we make a task for making sure that these tests are marked as integration tests and skipped by our build? 

At any rate, let's not have an entire chat session in our code. This should be removed as soon as we have a solution.

Reply:
actually I updated the assertions in this pull request and change == to >= comparisons. So adding new labels in wikidata will not break the tests anymore. It is very unlikely that the labels will be removed from wikidata, even if this is theoretically possible. ... For now I left the comments with the test code as I aggree that this is not the perfect solution. The separation of unit and integration tests should be addressed in another ticket.

Reply:
PS: It would be fine to create a new issue by copy-pasting the chat from the code comments.